onboarding.md

```diff
@@ -37,287 +36,12 @@ A member of the @niteoweb/peopleops team will do the following to start the onbo
-* [ ] Access to Heroku apps.
-* [ ] Join NiteoWeb Team on GitHub.
-* [ ] ZenHub Team invitation.
-* [ ] Invite to donations spreadsheet.
-* [ ] Configure Resilio Sync.
-```
-
-# [Email Template] First Email
-
-```
-Hey!
-
-Welcome to Niteo! We keep everything related to work on GitHub so I will be short:
-* read https://github.com/niteoweb/handbook/blob/master/onboarding.md,
-* open <LINK TO ONBOARDING ISSUE> and start at the top.
-
-
-Regards
-```
-
-
-
-# [Email Template] Niteo Email Account
-
-```
-Hey!
-
-Here's your Niteo email. You'll be using it for communication with us and for subscribing to websites.
-
-w: https://apps.rackspace.com/
-u: [email]
-p: [pass]
-
-Please change your password immediately after you login. You can do that in the top right Settings menu.
-
-Configure the email on your desktop app, find the instructions here: https://emailhelp.rackspace.com
-
-The next thing that you need to do is set up S/MIME email encryption. Generate a Comodo Certificate at https://www.comodo.com/home/email-security/free-email-certificate.php. You’ll need to Google on how to set encryption on your desktop app.
-
-This email is signed with my cert, you should be able to reply with an encrypted message, signed with your cert so that I can then reply back with an encrypted message as well.
-
-Once you have encryption ready, I’m sending over more information.
-
-
-Regards
-```
-
-
-# [Email Template] OpenVPN Access
-
-```
-Hey!
-
-Now that encryption is configured, you need to configure OpenVPN so you can connect to our internal services.
-
-Configuration files are attached, normally you just double-click them and follow instructions.
-
-When you manage to connect to our VPN you will then be able to access our internal services, such as http://docs.niteoweb.com/.
-
-
-Regards
-```
-
-# Email Signatures
-
-A few examples to base your signature off of. Choose your own title, it can be whatever you wish.
-
-```
-Richard Hendricks
-
-CEO
----------------
-Niteo.co
-
-E: rh@niteo.co
-W: www.niteo.co
-```
-
-```
-Erlich Bachmann
-
-Backend Specialist
-------------------
-Niteo.co
-
-E: eb@niteo.co
-W: www.niteo.co
-```
-```
-Monica Hall
-
-Senior (con)Sultan
-------------------
-Niteo.co
-
-E: mh@niteo.co
-W: www.niteo.co
-```
-
-# Offboarding
+## Onboarding References
 
-```
-* [ ] Access to team 1Password.
-* [ ] Heroku access.
-* [ ] GitHub repositories access.
-* [ ] PyPICloud access.
-* [ ] Donations spreadsheet.
-* [ ] Slack access.
-* [ ] Email account.
-* [ ] Intra account.
-* [ ] VPN access.
-* [ ] Zoom.us account.
-* [ ] Sentry access.
-* [ ] EBN account.
-* [ ] DMON account.
-```
+ * [Welcome Email Template](templates/onboarding/welcome-email.md)
+ * [Niteo Account Email Template](templates/onboarding/niteo-account-email.md)
+ * [OpenVPN Email Template](templates/onboarding/openvpn-email.md)
+ * [Trialist Onboarding Issue Template](niteoweb/operations/.github/onboard-trial_tpl.md)
```

Comment:
All three ops templates need to included the correct link to open new ops issue with them